config/foreman.migrations/20201224125100_ansible_ssh_args.rb

```diff
@@ -0,0 +1,4 @@
+# migrate the default
+if answers['foreman_proxy']['ansible'].is_a?(Hash) && answers['foreman']['ansible']['ssh_args'] == '-o ProxyCommand=none'
```

Comment:
I think you want `answers['foreman_proxy::plugin::ansible'].is_a?(Hash) && answers['foreman_proxy::plugin::ansible']['ssh_args']` given that the class is `foreman_proxy::plugin::ansible` in the answers file.

The Katello and Foreman Proxy Content scenarios could also benefit from this migration.

Reply:
Done :+1: Thanks :)